README.md

```diff
@@ -19,14 +19,29 @@ VROOM can also solve any mix of the above problem types.
 
 ## Installation
 
-Pyvroom currently makes binaries for on MacOS and Linux.
+Pyvroom currently makes binaries for on MacOS and Linux (Windows is WIP).
 
 Installation should be as simple as:
 
 ```bash
 pip install pyvroom
 ```
 
+## Building
+
+Building the source distributions on another OS requires:
+- the `./build-requirements.txt` Python dependencies
```

Comment:
ah, didn't include that file yet, but would be helpful no?

Reply:
I added it

Reply:
I still think it can't hurt to have it here for the old people like me who are still used to `python setup.py xx`;) happy to remove it as well though